src/riak_core_handoff_sender.erl

```diff
@@ -484,9 +507,10 @@ send_objects(ItemsReverseList, Acc) ->
                         timer:now_diff(os:timestamp(), SyncClock) div 1000,
                         timer:now_diff(os:timestamp(), AckLogLast) div 1000]
                 ),
-                os:timestamp();
+                UpdConfigAcc = set_timeouts_and_thresholds(Acc),
+                UpdConfigAcc#ho_acc{acklog_last = os:timestamp()};
             {ok, _} ->
-                AckLogLast;
+                Acc;
```

Comment:
Why is this correct... it was AckLostLast from line 438 and you still have AckLogLast in line 461?

Reply:
Ah, got it, the type is now the complete Acc...